core/libp2p/peer/impl/identity_manager_impl.hpp

```diff
@@ -17,9 +17,9 @@ namespace libp2p::peer {
 
     explicit IdentityManagerImpl(crypto::KeyPair keyPair);
 
-    const peer::PeerId &getId() const noexcept override;;
+    const peer::PeerId &getId() const override;
 
-    const crypto::KeyPair &getKeyPair() const noexcept override;;
```

Comment:
You can leave them - it is possible to make exception guarantee more strict in the implementation, than in the interface

Reply:
noexcept plays badly with mocks, so I removed them